src/argilla_server/contexts/info.py

```diff
@@ -0,0 +1,60 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+
+import psutil
+
+from argilla_server._version import __version__
+from argilla_server.daos.backend import GenericElasticEngineBackend
+
+
+def argilla_version() -> str:
+    return __version__
+
+
+def elasticsearch_status() -> dict:
+    return GenericElasticEngineBackend.get_instance().client.get_cluster_info()
```

Comment:
Just to not reuse old code:

```suggestion 
def elasticsearch_status(search_engine: SearchEngine) -> dict:
    return search_engine.client.get_cluster_info()
```